bigchaindb/__init__.py

```diff
@@ -9,7 +9,7 @@
     'server': {
         # Note: this section supports all the Gunicorn settings:
         #       - http://docs.gunicorn.org/en/stable/settings.html
-        'bind': '0.0.0.0:9984',
+        'bind': 'localhost:9984',
```

Comment:
> Could you briefly comment on why this change is necessary?
Sure, you can take a look to the changes I made [in the docs](https://github.com/bigchaindb/bigchaindb/pull/183/files?short_path=af33c3d#diff-af33c3d6099d055484e6f6471b741cac)

> Is there a reason for not parametrizing the host?
Yes, the `server` section is compatible to the [Gunicorn settings](http://docs.gunicorn.org/en/stable/settings.html), so I'm just using their stuff. I thought about splitting `host` and `port` but this would require to add code. In this case I preferred to keep things simple and compatible with Gunicorn. This information is in the `bigchaindb/__init__.py` file but not in the docs, I'm adding it now!